README.md

```diff
@@ -63,3 +63,5 @@ Running this configuration, you'll be asked for github token and then e2e-demos
 If you want to run/debug different suite, change `-ginkgo.focus` parameter in `.vscode/launch.json`.
 # Load tests 
 [Refer this](https://github.com/redhat-appstudio/e2e-tests/blob/main/docs/LoadTests.md)
+
+JIRA Issue: https://issues.redhat.com/browse/HACBS-834
```

Comment:
Why is this needed here?

Reply:
Pavel Sturc suggested I put the Jira link in the readme file

Reply:
Fixed
Please approve & lgtm my PR